wasi/wasi.go

```diff
@@ -143,7 +143,24 @@ type API interface {
 	// TODO: ProcExit
 	// TODO: ProcRaise
 	// TODO: SchedYield
-	// TODO: RandomGet
+
+	// RandomGet is a WASI function that write random data in buffer (rand.Read()).
+	//
+	// * buf - is a offset to write random values
+	// * bufLen - size of random data in bytes
+	//
+	// For example, if `HostFunctionCallContext.Randomizer` initialized
```

Comment:
this drifted and also we are probably being too specific on the mechanics of random.

maybe `For example, if bufLen = 5, we expect `ctx.Memory.Buffer` to contain 5 random bytes:`

Reply:
@codefromthecrypt 
You mean not to use in comments concrete random bytes from specified seed?
And also not to check concrete random bytes form seed, cause crypto/rand non-deterministic and we can't seed it, right? 
Just check in test that memory is filled

Reply:
no I mean in this godoc we don't need to get into how secure random works. I still like that you are using tests that are resulting in a known value

Reply:
or we can seed math/rand from crypto/rand, like this
```
var seed int64
binary.Read(crypto_rand.Reader, binary.BigEndian, &seed)
math_rand.NewSource(seed)
```
and still use simple seed and check concrete bytes in tests

Reply:
> no I mean in this godoc we don't need to get into how secure random works. I still like that you are using tests that are resulting in a known value

got it, thanks for your feedback